validation.js

```diff
@@ -0,0 +1,17 @@
+const error = document.querySelector('.error');
+const email = document.querySelector('#email');
+const submit = document.querySelector('#submit');
+
+const checkLowerCase = () => {
+  if (email.validity.patternMismatch) error.textContent = 'The form has not been submitted. Please type email iin lowercase';
+  else error.textContent = '';
+  return !email.validity.patternMismatch;
+};
+
+submit.addEventListener('submit', () => {
+  checkLowerCase();
+});
+
+email.addEventListener('input', () => {
+  checkLowerCase();
+});
```

Comment:
While you made a good job implementing the features, however, most of them are not addressed yet. Let's fix them before moving to the next task.

- [ ] The contact form contains at least one validation checking if the email is written in all lowercase:
- If a user submits the contact form with the email field containing uppercase characters an error message should show up.
- If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

- [ ] The error message should show up close to the "submit" button.
- [ ] The error message fits the design from the chosen template in Figma (e.g., font, colors, images, tests, margins).
- [ ] The form "submit" or the submit button/input "click" event is used to attach the function checking the email format.
- [ ] The students can check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly.

These features of your contact form are not working properly. Kindly, fix it!